packages/driver/src/cypress/commands.js

```diff
@@ -40,6 +40,11 @@ const builtInCommands = [
   require('../cy/net-stubbing').addCommand,
 ]
 
+const reservedCommandNames = [
+  'getAlias',
+  'reset',
+]
```

Comment:
This is a _super_ naive first take. Is there a better way for us to enumerate potential collisions - maaaybe even something we could use to pull into the docs automagically as well? 🤞